setup.py

```diff
@@ -20,6 +23,28 @@
 ]
 
 
+class BuildPyCommand(setuptools.command.build_py.build_py):
+    """Custom build command."""
+
+    def run(self):
+        print('Downloading UI static files')
+
+        download_url = 'https://github.com/ansible/ansible-hub-ui/' + \
```

Comment:
Please move `download_url` outside the function (class level \ module level) and make it a constant.